### PR TITLE
Add a section about collaboration

### PR DESCRIPTION
**Pull request by johndbritton**
Please review and let me know what you think

### CODE REVIEW

**Comment by edwfeng** [comment]
looks good

**Comment by bca-benise** [comment]
Probably a bad idea.

File: README.md [hunk 1]
```diff
@@ -20,6 +20,12 @@ def world
   puts "hello"
 end
 
+# Collaboration
+* Make a branch
+* Push to GitHub
```
Comment (johndbritton) [inline README.md]: @edwfeng Am I missing anything ❓

**Comment by gman1230321** [comment]
cool

**Pull request merged**